books/free-programming-books-langs.md

```diff
@@ -1164,6 +1164,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 * [Angular 1 Style Guide](https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md) - John Papa (HTML)
 * [Angular for the jQuery developer](http://www.ng-newsletter.com.s3-website-us-east-1.amazonaws.com/posts/angular-for-the-jquery-developer.html) - ng-newsletter, Ari Lerner, David Wu, Nate Murray (HTML)
+* [Angular Testing Succinctly](https://www.syncfusion.com/succinctly-free-ebooks/angular-testing-succinctly) - Joseph D. Booth (HTML, PDF, EPUB, [Companion Repo and example codes](https://github.com/SyncfusionSuccinctlyE-Books/Angular-Testing-Succinctly)) (email address required for EPUB and PDF)
```

Comment:
Companion repo is mentioned in the book?

Reply:
Yes, it is mentioned in the GitHub Repo and the book's official page as source code.

Reply:
Then you can remove it

Reply:
Ok, I have removed it.